config/locales/inherited_proofing/es.yml

```diff
@@ -0,0 +1,16 @@
+---
+es:
+  inherited_proofing:
+    buttons:
+      continue: replaceme
```

Comment:
Will we create a ticket to replace these, if one doesn't already exist? Asking since I recall instances in the past where placeholder content persisted longer than we had intended it to.

Reply:
There's already a ticket and it's in this new spring! ✌️

Reply:
Yes, there is a follow-on story (in the next sprint) for adding the translations. Our team is figuring out the required stories and ordering. Requiring the translations up-front, is kind-of a pain. Maybe the answer (in this case) is to create the page without any translation content (including English, ugg!).